modules/btcd/btcd_wrapper/wrapper.go

```diff
@@ -281,4 +282,14 @@ func BTCDAddress(data C.ByteArray) *C.char {
 	return C.CString(prefix + addr.EncodeAddress())
 }
 
+//export BTCDBip32MasterKeygen
+func BTCDBip32MasterKeygen(data C.ByteArray) *C.char {
+	seed := C.GoBytes(unsafe.Pointer(data.data), data.length)
+	masterKey, err := hdkeychain.NewMaster(seed, &chaincfg.MainNetParams)
+	if err != nil {
+		return C.CString(fmt.Sprintf("ERROR: %v", err))
+	}
+	return C.CString(masterKey.String())
+}
+
```

Comment:
We should skip this case because other implementations allow it. Even though btcd validates seed length (128-512 bits), we want to test that inputs accepted by other implementations execute correctly.

```suggestion
//export BTCDBip32MasterKeygen
func BTCDBip32MasterKeygen(data C.ByteArray) *C.char {
	seed := C.GoBytes(unsafe.Pointer(data.data), data.length)
	masterKey, err := hdkeychain.NewMaster(seed, &chaincfg.MainNetParams)
	if err != nil {
		// Skip seed length validation errors (128-512 bits requirement) as other
		// implementations don't enforce this constraint.
		if err.Error() == "seed length must be between 128 and 512 bits" {
			return nil
		}
		return C.CString("")
	}
	return C.CString(masterKey.String())
}
```